lib/main.dart

```diff
@@ -242,10 +242,8 @@ class _FlutterScreenState extends State<FlutterScreen> {
   Future<void> updateParticipantsList() async {
     try {
       var conference = await dolbyioCommsSdk.conference.current();
-      var participantsList =
-          await dolbyioCommsSdk.conference.getParticipants(conference);
-      final availableParticipants = participantsList
-          .where((element) => element.status != ParticipantStatus.left);
+      var participantsList = await dolbyioCommsSdk.conference.getParticipants(conference);
+      final availableParticipants = participantsList .where((element) => element.status != ParticipantStatus.left);
```

Comment:
I was wondering though if we should use Flutter format for this example. It will probably make the code look more "normal" for a Flutter developer. 
In this case it would have probably moved the .`where ...` on the next line (as it was) as it enforces a line of no more than 80 chars.